e/egeria/egeria_ubuntu_20.04.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : egeria
+# Version       : egeria-release-2.10 
+# Source repo   : https://github.com/odpi/egeria
+# Tested on     : Ubuntu 20.04.1 LTS (Focal Fossa)
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Nagesh Tarale <Nagesh.Tarale@ibm.com>
```

Comment:
Pl. list in the comments section, what all repos from wkc jar sheet will be built using this script

Reply:
updated

Reply:
You need to put the list in script for any user to refer. Pl. refer to s/spring-boot-starter/spring-boot-generic_ubuntu_20.04.sh